src/main/java/com/joss/conductor/mobile/annotation/AndroidId.java

```diff
@@ -0,0 +1,17 @@
+package com.joss.conductor.mobile.annotation;
+
+import java.lang.annotation.*;
+
+/**
+ * Created on 3/14/18.
+ */
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.FIELD)
+public @interface AndroidId {
```

Comment:
I'd suggest changing the name to `AndroidFindBy`, just so that it more explicit what it's doing. 

`@AndroidId(xpath = "//path/to/thing")` becomes `@AndroidFindBy(xpath = "//path/to/thing")` which is more intuitive to me, then you also have the `by` in the name which makes it a bit more explicit that it's for use with the `By` class.

Reply:
I was concerned people might confuse this and the `@AndroidFindBy` from Appium.  I can definitely change it if you don't think that'll be an issue.

Reply:
Changed it to `AndroidFindBy` for now.